adcm_client/objects.py

```diff
@@ -937,8 +937,16 @@ def _upload(self, bundle_stream) -> Bundle:
         return self.bundle(bundle_id=data['id'])
 
     @allure_step('Upload bundle from "{1}"')
-    def upload_from_fs(self, dirname) -> Bundle:
-        return self._upload(stream.file(dirname))
+    def upload_from_fs(self, dirname, **args) -> BundleList:
+        streams = stream.file(dirname, **args)
+        result = None
+        if isinstance(streams, list):
+            for st in streams:
+                result = BundleList(self._api, id=self._upload(st).id) if result is None \
```

Comment:
There is no needs in that code, if you initialize result as empty BundleList, and not None.

Reply:
Fixed